docs/apis-hook.md

```diff
@@ -4,4 +4,149 @@ sidebar_position: 4
 
 # APIs hook
 
-This page will discuss - How to interact with the ReactPlay data using custom hook.
+In React Play, we work with APIs by using our custom-made react hooks. They are as follows:
+
+1. [useCacheResponse](#usecacheresponse)
+1. [useContributors](#usecontributors)
+1. [useFeaturedPlays](#usefeaturedplays)
+1. [useFetch](#usefetch)
+1. [useGetPlays](#usegetplays)
+1. [useGitHub](#usegithub)
+1. [useLikePlays](#uselikeplays)
+1. [useLocalStorage](#uselocalstorage)
+
+In this section, we will discuss each of them in detail.
+
+## useCacheResponse
+
+As the name suggests, this hook is used to create and maintain a cached version of the data.
```

Comment:
Can we have a line of when should we use it. @Angryman18 may help getting you more details about it.

Reply:
yup. as the same suggests, this hook can be used create temporary cache of any kind of data in our codebase (not on browser).

Reply:
I have incorporated this change.